lib/src/test/kotlin/starknet/utils/DevnetClient.kt

```diff
@@ -77,7 +77,7 @@ class DevnetClient(
                 "--seed",
                 seed.toString(),
                 "--sierra-compiler-path",
-                "../cairo/target/debug/starknet-sierra-compile",
+                "src/test/resources/cairo_compilers_v2/cairo/bin/starknet-sierra-compile",
```

Comment:
this path could be simplified to something like

`src/test/resources/compilers/v2/cairo/bin`